### PR TITLE
Transformer returns always transformed code if file contains only unused linaria code

### LINKED ISSUE #624: Transforming file contains only unused linaria code always results in original code
## Environment
- Linaria version: 2.0.0-alpha.5

## Description
File with unused code like:
```js
import { css } from "linaria";

const bar = css`
  background-color: red;
`;
```
Should be transformed to sth like:
```js
import { css } from "linaria";

const bar = 'className'
```
Then bundler should remove it later anyway using tree shaking.
If code wasn't transformed then bundler will include original implementation with error in runtime because css were treated as a function call( which can include side effects).

## Possible solution
Always return transformed code from `transformer` instead of original one

### PR DESCRIPTION
**Pull request by TMaszko**
## Motivation
There was a bug with transforming file that contains only unused linaria code( like variables that never were used again)

## Summary
Fixes #624
Transformer returns

## Test plan
Added tests for this case.

### CODE REVIEW

File: src/__tests__/transform.test.ts [hunk 1]
```diff
@@ -212,3 +212,24 @@ it("doesn't throw due to duplicate preset", async () => {
     )
   ).not.toThrow('Duplicate plugin/preset detected');
 });
+
+it('testowy', async () => {
```
Comment (jayu) [inline src/__tests__/transform.test.ts:216]: Please adjust the name of the test

**Comment by jayu** [comment]
Regarding the TS error, I had the same once when I run typecheck locally, but it didn't happen on CI, so I ignored it. Do you have this error locally also?

**Comment by TMaszko** [comment]
Yeah I had it

File: src/__tests__/transform.test.ts (hunk 1 above)
Comment (TMaszko) [reply src/__tests__/transform.test.ts:216]: 😂

**Pull request merged**